client/my-sites/email/email-management/style.scss

```diff
@@ -120,18 +122,34 @@
 
 .email-list-inactive {
 	margin-top: 15px;
+}
 
-	> .card {
```

Comment:
This rule would also target the `SectionHeader` in `EmailListInactive`, which it shouldn't. That's why I removed it and changed around the styling a little bit.